chain/db/request.go

```diff
@@ -169,12 +170,16 @@ func (b *BandDB) handleMsgRequestData(
 	if err != nil {
 		return err
 	}
+	request, err := b.OracleKeeper.GetRequest(b.ctx, types.RequestID(id))
```

Comment:
Use `oracle.RequestID`

Reply:
but we have 2 place to add new request (ibc.go), so in my opinion change this on `AddNewRequest` function instead